stonesoup/dataassociator/general.py

```diff
@@ -0,0 +1,162 @@
+from typing import Tuple, Collection
```

Comment:
This file needs adding to docs/source/stonesoup.dataassociator.rst so that modules appear in the docs

Reply:
Good spot